src/libs/actions/Session.js

```diff
@@ -208,19 +208,20 @@ function restartSignin() {
  *
  * @param {String} password
  * @param {String} validateCode
+ * @param {String} email
  */
-function setPassword(password, validateCode) {
+function setPassword(password, validateCode, email = '') {
     Onyx.merge(ONYXKEYS.ACCOUNT, {error: '', loading: true});
 
     API.SetPassword({
-        email: credentials.login,
+        email: email || credentials.login,
         password,
         validateCode,
     })
         .then((response) => {
             if (response.jsonCode === 200) {
-                const {authToken, email} = response;
-                createTemporaryLogin(authToken, email);
+                const {authToken, login} = response;
```

Comment:
Curious why we are changing this from `email` to `login`?
What response params does the API for `SetPassword` send back?
Are we modifying them in the Web-E PR?

Reply:
@marcaaron There's another variable called `email` in the upper scope, so `eslint` points it out as an error.

Reply:
Makes sense. So, is there a `login` property on the `response`? 
If so then we can [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring). If not, `login` will be `undefined`.

Reply:
Ahh you'd definitely be correct in saying that it would be `undefined`. Auth actually sends back the email field, that's totally my bad! I can refrain from using object destructuring here which should solve the problem.